libs/newsletter-workflow/src/lib/steps/newsletterData/thrasherLayout.ts

```diff
@@ -19,8 +19,6 @@ Should the single thrasher appear on app and web?
 
 Do you need one or more multi-thrashers i.e. sets where the thrasher for **{{name}}** is combined with thrashers for 2 or more other newsletters?
 
-***NEED TO DETERMINE THE BEST WAY TO CAPTURE THIS INFORMATION - CLEARLY A CHECKBOX IS INSUFFICIENT!***
```

Comment:
This was just a holding comment to remind us that the page was incomplete - but there is now a separate trello ticket for capturing this information